pulsar-proxy/src/main/java/org/apache/pulsar/proxy/server/ProxyConfiguration.java

```diff
@@ -35,6 +35,14 @@
     // ZooKeeper session timeout
     private int zookeeperSessionTimeoutMs = 30_000;
 
+    // If Discovery Service is Disabled the proxy will just authenticate the client 
+    // and forward all requests to a VIP or any other service discovery port 
+    private boolean discoveryServiceEnabled = true;
```

Comment:
I think we might not need this flag? If global-zk address is not provided then proxy should fall to provided discovery-url? similar as what we have done for websocket `WebSocketProxyConfiguration.java`?